security-analysis/src/test/java/com/powsybl/security/SecurityTest.java

```diff
@@ -99,22 +99,25 @@ public void printPostContingencyViolationsWithPreContingencyViolationsFiltering(
                                  "Contingency,Status,Action,Equipment,Violation type,Violation name,Value,Limit,Loading rate %",
                                  "contingency1,converge,,,,,,,",
                                  ",,action2,,,,,,",
-                                 ",,,line2,CURRENT,10',950.000,900.0,106"),
+                                 ",,,line2,CURRENT,10',950.0000,900.0000,105.5556"),
                      writer.toString().trim());
     }
 
     @Test
-    public void printLimitsViolations() {
-        assertEquals(String.join("\n",
-                                 "+---------+--------------+---------------+----------------+----------------+--------+--------+------------------+----------------+",
-                                 "| Country | Base voltage | Equipment (2) | Violation type | Violation name | Value  | Limit  | abs(value-limit) | Loading rate % |",
-                                 "+---------+--------------+---------------+----------------+----------------+--------+--------+------------------+----------------+",
-                                 "|         |              | line1         | CURRENT        | 20'            | 1100.0 | 1000.0 | 100.0            | 110            |",
-                                 "|         |              | line2         | CURRENT        | 10'            | 950.0  | 900.0  | 50.0             | 106            |",
-                                 "+---------+--------------+---------------+----------------+----------------+--------+--------+------------------+----------------+"),
+    public void printLimitsViolations() throws Exception {
+        Locale tmpLocale = Locale.getDefault();
+        Locale.setDefault(Locale.FRANCE);
```

Comment:
It’s forbidden to change the locale like this because in multi-thread mode, you can change the result of another unit test. You should use a custom FormatterConfig